editors/kakoune/Portfile

```
@@ -0,0 +1,25 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                kakoune
+version             2020.09.01
+github.setup        mawww kakoune ${version} v
+categories          editors
+platforms           darwin
+license             Unlicense
```

"Unlicense" is not a license I was familiar with, and it is not a name known to the MacPorts license-checking infrastructure, but the first line of the license reads:

> This is free and unencumbered software released into the public domain.

hence please change this the corresponding value that MacPorts does recognize:

```suggestion
license             public-domain
```

---

I think they are slightly different from a legal perspective, but changed as requested.